packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -118,8 +118,9 @@ module.exports = {
     'react/jsx-first-prop-new-line': ['error', 'multiline-multiprop'],
     'react/jsx-equals-spacing': ['error', 'never'],
     'react/jsx-indent': ['error', 2],
-    'react/no-did-mount-set-state': 'error',
-    'react/no-did-update-set-state': 'error',
+    // Off as of React 16.3, see https://github.com/yannickcr/eslint-plugin-react/issues/1754
+    'react/no-did-mount-set-state': 'off',
+    'react/no-did-update-set-state': 'off',
```

Comment:
Is there a reason these need to be set to `off` and not just removed?

Reply:
No. I thought we extend the recommended config, but since we didn't we can simply remove them :-)